indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -1146,6 +1141,21 @@ private void taskComplete(
     return ImmutableList.copyOf(getWorkerFromZK(lazyWorkers.values()));
   }
 
+  public static Collection<ImmutableWorkerInfo> getImmutableWorkerFromZK(Collection<ZkWorker> workers)
```

Comment:
I think this can be private. Also it seems that its only use immediately makes an ImmutableList, so maybe it'd be better for this method to just return an ImmutableList.